### PR TITLE
build: bump mkdocs-material -> 9.0.6 + fixes

### PR DESCRIPTION
**Pull request by Valastiri**
## Summary
Updates our theme to the latest version with the new improved search function from material. Some fixes are applied as part of this PR:

- Fix use cases of `!!! error` admonition as this seems to be deprecated upstream.
- texture-changes.md minor correction of header capitilization
- move the vnav tag in vnav symbology pages to parent overview page
- replace video iframe in vnav docs to plugin shortcode ![video-embed]
- removed outdated larger-admon stylesheet 
- **clamps search results to 5 lines via CSS** (this helps alleviate issues with pages that contain large amounts of markdown table data to overtake search results)
- **Remove** `exclude-search` plugin - see [issue comment below](https://github.com/flybywiresim/docs/pull/692#issuecomment-1374764063) for more info.

### Location
- requirements.txt
- various pages to provide fixes

Discord username (if different from GitHub): Valastiri#8902

### CODE REVIEW

**Comment by pdellaert** [comment]
Feedback on Bot Search functionality: 

TL;DR: Things work as expected. But search results is a bit different (which is probably to be expected with a new implementatoin)

**CURRENT SITE**

Screen Shot 2022-12-10 at 11 34 13 AM
Screen Shot 2022-12-10 at 11 34 20 AM

Screen Shot 2022-12-10 at 11 34 35 AM


**NEW SITE**

Screen Shot 2022-12-10 at 11 35 01 AM
Screen Shot 2022-12-10 at 11 35 08 AM

Screen Shot 2022-12-10 at 11 35 21 AM

**Comment by Valastiri** [comment]
Utilizing search.boost should clean up weird search results when the terminology is referenced a lot. Most notable on pages that is primarily a markdown table.

image

**Comment by Valastiri** [comment]
Will probably need to deprecate the `exclude-search` plugin. Tested by searching `Autopilot` yields a result for the page that is not yet built but included in the docs/ directory. 

Example:

image

Previously on 8.x.x searching `autopilot` would not yield a result with our current plugin. 

Material now has it built in via front matter per page  -> https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#search-exclusion

This would probably be preferrable as anyone writing docs for large scale features in aircraft can leave incomplete pages in the directory without them being searchable. 

```yml
  exclude-search:
    exclude:
      - pilots-corner/advanced-guides/flight-planning/fplan.md
      - pilots-corner/advanced-guides/flight-planning/offset.md
      - pilots-corner/advanced-guides/flight-planning/overview.md
      - pilots-corner/advanced-guides/flight-guidance/approaches.md
      - pilots-corner/advanced-guides/flight-guidance/autopilot.md
      - pilots-corner/advanced-guides/flight-guidance/lnav.md
  ``` 

The above pages all exist within our production built. Setting the search exclusion on the page itself would prevent issues with maintainers or PR authors forgetting to remove the exclusion from the config (was an issue with a few pages before).

## To Do

- [x] Migrate search exclusion to the pages listed above via new built-in method
- [x] Remove exclude-search from config
- [x] Remove plugin from requirements.txt
- [x] Update documentation project page with short example of how to set search exclusion

**Comment by Valastiri** [comment]
check build
